### PR TITLE
add many metrics

### PR DESCRIPTION
**Pull request by zhangxu19830126**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:
Add many metrics

### CODE REVIEW

**Review by daviszhen — APPROVED** [review]

File: pkg/sql/compile/compile.go [hunk 1]
```diff
@@ -379,6 +380,9 @@ func (c *Compile) run(s *Scope) error {
 
 // Run is an important function of the compute-layer, it executes a single sql according to its scope
 func (c *Compile) Run(_ uint64) (*util2.RunResult, error) {
+	start := time.Now()
+	defer v2.SQlRunDurationHistogram.Observe(time.Since(start).Seconds())
```
Comment (nnsgmsone) [inline pkg/sql/compile/compile.go:384]: 这个看着是有锁的。。。这个对应并发高的情况不会造成瓶颈吗？

Comment (nnsgmsone) [reply pkg/sql/compile/compile.go:384]: 搞错了。。。。这个是我们自己的那个Obserce才有锁。。。

**Review by nnsgmsone — APPROVED** [review]

**Review by aylei — APPROVED** [review]
lgtm

**Review by aptend — APPROVED** [review]

**Review by XuPeng-SH — APPROVED** [review]

**Review by qingxinhome — APPROVED** [review]

**Review by xzxiong — APPROVED** [review]
lgtm

**Review by reusee — APPROVED** [review]

**Pull request merged**